src/docs/PSA/references/reference.adoc

```diff
@@ -418,6 +418,47 @@ This service allows to deploy through ProActive Service Automation (PSA) Portal
 
 NOTE: More details about the variable description can be found https://hub.docker.com/_/postgres/[here^].
 
+
+=== Greenplum
+
+This service allows to deploy through ProActive Service Automation (PSA) Portal a Greenplum Database server. The service can be started using the following variables.
```

Comment:
This service allows to deploy -> This service deploys
The service can be started using the following variables. -> The service takes as input the following variables.

Reply:
Thank you Nebil for the review. 
I need to apply your changes to all PSA (15 services) right ?